CMakeLists.txt

```diff
@@ -54,6 +54,13 @@ add_subdirectory(remote-processor)
 
 add_subdirectory(remote-process)
 
+# todo: Can those valgrind flags be moved in CTestCustom.cmake ?
```

Comment:
This is worse trying, no? Isn't it strange to have tests here at the base level?